art/art.py

```diff
@@ -552,24 +554,29 @@ def get_font_dic(font_name):
 
     :param  font_name: font's name
     :type font_name:str
-    :return font's dictionary
+    :return: font's dictionary
     """
     return FONT_MAP[font_name][0]
 
 
-def decor(decoration, reverse=False):
+def decor(decoration, reverse=False, both=False):
     """
     Return given decoration part.
 
     :param  decoration: decoration's name
     :type decoration:str
     :param reverse: true if second tail of decoration wanted
     :type reverse:bool
-    :return decor's tail
+    :param both: both tails returning flag
+    :type bool: bool
+    :return: decor's tail or tails as str or list
     """
     if isinstance(decoration, str) is False:
         raise artError(DECORATION_TYPE_ERROR)
+    decoration = decoration.lower()
     decoration = indirect_decoration(decoration)
+    if both is True:
+        return DECORATIONS_MAP[decoration]
     if reverse is True:
         return DECORATIONS_MAP[decoration][-1]
     return DECORATIONS_MAP[decoration][0]
```

Comment:
how should we handle `both=True, reverse=True`, here we are ignoring `reverse` but isn't it better if we raise a warning for this situation?

Reply:
I think it's not a big deal.